integration/integration_test.go

```diff
@@ -3376,90 +3313,89 @@ func (s *IntSuite) TestPAM(c *check.C) {
 	}
 
 	for _, tt := range tests {
-		// Create a teleport instance with auth, proxy, and node.
-		makeConfig := func() (*check.C, []string, []*InstanceSecrets, *service.Config) {
-			tconf := s.defaultServiceConfig()
-			tconf.Auth.Enabled = true
-
-			tconf.Proxy.Enabled = true
-			tconf.Proxy.DisableWebService = true
-			tconf.Proxy.DisableWebInterface = true
-
-			tconf.SSH.Enabled = true
-			tconf.SSH.PAM.Enabled = tt.inEnabled
-			tconf.SSH.PAM.ServiceName = tt.inServiceName
-			tconf.SSH.PAM.UsePAMAuth = tt.inUsePAMAuth
-			tconf.SSH.PAM.Environment = tt.environment
-
-			return c, nil, nil, tconf
-		}
-		t := s.newTeleportWithConfig(makeConfig())
-		defer t.StopAll()
+		t.Run("", func(t *testing.T) {
```

Comment:
Nit: Can you add description?

Reply:
Added desc based on test comments - assuming they are accurate.